migrate-base.js

```diff
@@ -142,7 +142,11 @@ async function migrateBase ({ web3, spinner, confirm, opts, logTx, previousMigra
   if (Number(arcVersion.slice(-2)) >= 22) {
     await deploy(require('@daostack/arc/build/contracts/Redeemer.json'))
   }
-  await deploy(require('@daostack/arc/build/contracts/UGenericScheme.json'))
+  if (Number(arcVersion.slice(-2)) >= 24) {
```

Comment:
why this is needed ?
is that needed for each new arc version ?

Reply:
Just when we have breaking changes